plugins/source/aws/resources/services/ec2/launch_template_versions_fetch.go

```diff
@@ -0,0 +1,28 @@
+package ec2
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchEc2LaunchTemplateVersions(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	var config ec2.DescribeLaunchTemplateVersionsInput
```

Comment:
I think this was missing a config parameter to specify the launch template ID from the parent, so I pushed up a fix - let me know if you think that wasn't right